index.html

```diff
@@ -880,6 +880,67 @@ <h2>
 
 </pre>
     </section>
+    <section class='informative'>
+      <h2>
+        Privacy and Security Considerations
+      </h2>
+      <p>
```

Comment:
So, I tried doing exactly this format in the past (in a different spec) and people complained. Maybe we can just chop this down to the "yes" stuff, and make a paragraph out of it.

Reply:
It's pretty useful to have the positive and negative aspects covered, but I agree that it should be readable prose rather than answers to questions whether they are relevant or not...